sig/lrama/states.rbs

```diff
@@ -0,0 +1,143 @@
+module Lrama
+  # States is passed to a template file
```

Comment:
It seems these comments come from "lib/lrama/states.rb". Are they needed for rbs file?

Reply:
Comments were included because they were generated by rbs prototype and committed without change.
I have deleted comments that seem unnecessary.